superbench/runner/runner.py

```diff
@@ -233,6 +235,17 @@ def fetch_results(self):    # pragma: no cover
             )
         )
 
+    def __signal_handler(self, signum, frame):
+        """Signal handler for runner.
+
+        Args:
+            signum (int): Signal number.
+            frame (FrameType): Timeout frame.
+        """
+        logger.info('Killed by %s, exiting ...', signal.Signals(signum).name)
```

Comment:
Do we need to check signum?

Reply:
only SIGINT and SIGTERM are trapped
https://github.com/microsoft/superbenchmark/blob/9a6dc23fed01896f4e265f2b677aa400b57be7b8/superbench/runner/runner.py#L463-L464

Reply:
I mean whether we should check here. For example, someone change the code in runner but forget here. It will trigger the non-expected issue.

Reply:
sure, updated